Code/Framework/AzToolsFramework/AzToolsFramework/Entity/PrefabEditorEntityOwnershipInterface.h

```diff
@@ -19,11 +19,31 @@
 
 namespace AzToolsFramework
 {
+    namespace Prefab
+    {
+        //! The RootAliasPath class stores an alias path that starts from the Prefab EOS root instance.
```

Comment:
nit: It's not a class and it doesn't store anything. It's only an alias

Reply:
Well admittedly it's an alias to AZ::IO::Path, which is a class that stores an alias.

Reply:
I've reworded this slightly to be more generic:
```
//! A RootAliasPath can be used to store an alias path that starts from the Prefab EOS root instance.
//! The root instance itself is included in the path. These can be used as Instance handles across systems
//! that do not have visibility over InstanceOptionalReferences, or that need to store Instance handles
//! for longer than just the span of a function without the risk of them going out of scope.
```